src/org/salesforce/apexdoc/ApexDoc.java

```diff
@@ -210,23 +213,32 @@ public static ClassModel parseFileContents(String filePath) {
                 // gather up our comments
                 if (strLine.startsWith("/*")) {
                     commentsStarted = true;
+                    if(strLine.startsWith("/**")){
+                    	lstComments.add(strLine);
```

Comment:
if the line ends with */, your change has already added it to lstComments before the */ is removed.